docs/src/understand/digital-twin.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Digital Twin
+tags: [Concept]
+sidebar_position: 2
+---
+
+# Digital Twin
+
+Thin-edge gives the tools to build
+a remote *digital twin* representation of a piece equipment
+as well as a *local representation* of its architecture,
+both being used to direct operations to and collect data from
+the key components of the equipment, being hardware, software or data.
+
+This digital representation gives an actionable view to the remote user
+as well as to the device operator and to local processes.
+
+- A cloud operator uses the digital twin to monitor, operate and manage the device,
+  visualizing on his user interface the telemetry data collected from the device components
+  and triggering operations on these remote components from his user interface.
```

Comment:
Minor wording change as the `his user interface` was repeated multiple times, and using `their` instead of `his`.

```suggestion
- A cloud operator uses the digital twin to monitor, operate and manage the device,
  visualizing the telemetry data collected from the device components
  and triggering operations on these remote components from their user interface.
```

Reply:
Fixed